### PR TITLE
working with just seconds and keygen accepting it is not necessary the timedelta conversion any more

### PR DESCRIPTION
**Pull request by jeffctr**

### CODE REVIEW

File: src/ssh_cert_service/utils/common.py [hunk 1]
```diff
@@ -1,32 +1,26 @@
 import re
+from datetime import timedelta, datetime
```
Comment (gweis) [inline src/ssh_cert_service/utils/common.py]: unused imports

File: tests/test_common.py [hunk 2]
```diff
@@ -1,23 +1,45 @@
 import pytest
+from datetime import timedelta
 
 VALIDITY = "+1d"
```
Comment (gweis) [inline tests/test_common.py]: unused constant

**Comment by gweis** [comment]
9 of 25 tests are failing

**Comment by gweis** [comment]
still 9 of 25 tests failing
I guess ssh-keygen doesn't like `-V -1:+1d`

**Comment by jeffctr** [comment]
@gweis what is weird I got `25 passed`

**Comment by gweis** [comment]
@jeffctr then I'd say your environment is different ....

one of the commands that fail during test runs:

```
# ssh-keygen -s test_ca.key -I TESTING -Z testing.com.au -V -1:+1d -n testing_user test.key.pub 
Invalid OpenSSH-format cipher 'testing.com.au'
```

We talked about the `-Z` before and that it was pure luck that it worked so far. Looking at man pages, I can't find where it's documented. What is `-Z` supposed to do? (openssh 8.6 uses it to select a cipher, and I think it's highly unlikely that `-Z` has been used for something different in previous versions).

**Comment by jeffctr** [comment]
@gweis  The `-Z` option restricts this certificate to a specific host within the domain. Probably I am working with a different one. I would remove that. 

https://man.openbsd.org/ssh-keygen.1#Z
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/sec-creating_ssh_ca_certificate_signing-keys

**Pull request merged**